GalacticArchive.IndexingEngine/initDbSchema.sql

```diff
@@ -1 +1,33 @@
-﻿
\ No newline at end of file
+﻿CREATE
```

Comment:
Not really mentioned in the pdf, but no need to include the create db statements in this file. The tests expect this file to just execute schema creation statements on a db. In most cases production apps also follow this pattern of not creating a db on the fly with an sql script file.